### ISSUE TITLE
Please actually implement skip_verify for archive.extracted

### ISSUE THREAD

**Issue by vernondcole** [saltstack/salt#34515]
### Description of Issue/Question

When I do not define an upstream hash for archive.extracted, Salt provides an error message suggesting that I use a "skip_verify" switch.

That switch does not exist.
### Setup

Virtualbox published an invalid checksum, so I want to ignore it this time...

``` (yaml)
vagrant-sdk:
  archive:
    - extracted
    - name: /opt/virtualbox/5.0.24
    - source: http://download.virtualbox.org/virtualbox/5.0.24/VirtualBoxSDK-5.0.24-108355.zip
    # - source_hash: http://download.virtualbox.org/virtualbox/5.0.24/MD5SUMS
    - archive_format: zip
    - user: {{ my_username }}
    - group: staff
    - trim_output: 5
    - if_missing: /opt/virtualbox/5.0.24/sdk/installer
```
### Steps to Reproduce Issue

`sudo salt-call --local state.sls {the state file we just defined}`

```
local:
----------
          ID: vagrant-sdk
    Function: archive.extracted
        Name: /opt/virtualbox/5.0.24
      Result: False
     Comment: Unable to verify upstream hash of source file http://download.virtualbox.org/virtualbox/5.0.24/VirtualBoxSDK-5.0.24-108355.zip, please set source_hash or set skip_verify to True
     Started: 16:33:26.085266
    Duration: 80.799 ms
```

But, attempting to define `skip_verify: true` is ineffective because the switch is not programmed.
### Versions Report

```
Salt Version:
           Salt: 2016.3.0-829-g69f8e3b

Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: 0.21.1
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.11+ (default, Apr 17 2016, 14:00:29)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.3
            ZMQ: 4.1.4

System Versions:
           dist: Ubuntu 16.04 xenial
        machine: x86_64
        release: 4.4.0-28-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by Ch3LL** [comment]
@vernondcole would you mind trying #34529 and seeing if that possible fix works for you?

**Comment by rallytime** [comment]
This was fixed in #34529 and is available in 2016.3.4 which is currently in the packaging process. Since we didn't hear back here, I am going to close this up. If this pops up again, let us know and we can take another look.

**Comment by mrichins** [comment]
I actually have been running into this problem recently. The lack of feedback stumped me for a while.
